### ISSUE TITLE
build fails on fresh install. trying restaurants app demo

### ISSUE THREAD

**Issue by hounsellal** [shoutem/ui#64]
Followed the instructions exactly. Clones shoutem/ui, cd'd into restaurants app and installed and linked. tried starting on iOS simulator and:

```
/Users/al/Dropbox/Development/chopchop/ui/examples/RestaurantsApp/node_modules/react-native/Libraries/WebSocket/RCTSRWebSocket.m:494:3: error: 
      ignoring return value of function declared with warn_unused_result
      attribute [-Werror,-Wunused-result]
  SecRandomCopyBytes(kSecRandomDefault, keyBytes.length, keyBytes.mutableBytes);
  ^~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/al/Dropbox/Development/chopchop/ui/examples/RestaurantsApp/node_modules/react-native/Libraries/WebSocket/RCTSRWebSocket.m:1334:5: error: 
      ignoring return value of function declared with warn_unused_result
      attribute [-Werror,-Wunused-result]
    SecRandomCopyBytes(kSecRandomDefault, sizeof(uint32_t), (uint8_t *)mask_key);
    ^~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2 errors generated.
```

And:

```
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/al/Dropbox/Development/chopchop/ui/examples/RestaurantsApp/ios/build/Build/Intermediates/RCTWebSocket.build/Debug-iphonesimulator/RCTWebSocket.build/Objects-normal/x86_64/RCTSRWebSocket.o RCTSRWebSocket.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Installing build/Build/Products/Debug-iphonesimulator/ShoutemRestaurants.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
Failed to install the requested application
An application bundle was not found at the provided path.
Provide a valid path to the desired application bundle.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/ShoutemRestaurants.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist
```

Any ideas?

**Comment by jackbillstrom** [comment]
Try `react-native upgrade`, and read this: https://github.com/facebook/react-native/issues/7308#issuecomment-219597774

**Comment by hounsellal** [comment]
Still doesn't work unfortunately.

**Comment by SoHotSoup** [comment]
Could you, please, try run XCode and clean your project (`Cmd+Shift+K`)? I've sometimes experienced this when trying to run `react-native run-ios` on my project regardless `@shoutem/ui`

**Comment by jkomyno** [comment]
Hi, I think this may help you @hounsellal : https://medium.com/@alberto.schiabel/react-native-on-xcode-beta-8-0-how-to-fix-initial-build-error-f0225c649850#.m0jxnivwt

**Comment by tenodi** [comment]
Closing, as this issue seems to be related with React Native.
